### ISSUE TITLE
PHP 8 support for v2.x

### ISSUE THREAD

**Issue by aimfeld** [laminas/laminas-cache#71]
While many laminas components seem to support PHP 8 by now, laminas-cache is currently preventing us from upgrading to PHP 8. I tried to require `"laminas/laminas-cache": "^3.0@dev"`, but `composer update` then fails with `laminas/laminas-cache-storage-adapter-apc 1.0.1 requires php ^5.6 || ^7.0`. I only need the redis adapter btw, but laminas-cache requires all adapters.

Is there an ETA for PHP 8 support or should I look for alternatives of laminas-cache?

**Comment by boesing** [comment]
This package and all adapters are not yet PHP 8 compatible (only unreleased, unstable 3.x).

You can try to ignore the fact but I encourage you not to use unreleased branches.

If you want to "drop" all adapters you do not need, you can add something like this to your `composer.json`:

```
{
    "replace": {
        "laminas/laminas-cache-adapter-blackhole": "*"
    }
}
```

This is possible since v2.10.
If you want to help with adding support for PHP 8 in the redis adapter, feel free to contribute. 

However, I dont think v3 will be ready somewhen in the near future as there is plenty of work left.

So you could help backporting PHP 8 support for 2.X aswell as redis Adapter 1.X.

Depending on what needs to be changed, I might be interested in creating another release.

**Comment by boesing** [comment]
Closed by merging #76
